tests/unit/broswe-storage-service.js

```diff
@@ -0,0 +1,37 @@
+describe('BrowserStorageService', function() {
+	var browserStorage;
+
+	beforeEach(module('ucApp'));
+
+	beforeEach(inject(function(_browserStorage_) {
+		// The injector unwraps the underscores (_) from around the parameter names when matching
+		browserStorage = _browserStorage_;
+	}));
+
+	describe('browserStorage', function() {
+		it('should set and retrieve a string', function() {
+			browserStorage.setItem('foo', 'bar');
+
+			expect(browserStorage.getItem('foo')).toEqual('bar');
+		});
+
+		it('should allow a specific value to be removed', function() {
+			browserStorage.setItem('foo', 'bar');
+			browserStorage.removeItem('foo');
+			expect(browserStorage.getItem('foo')).not.toEqual('bar');
```

Comment:
I would change your assertion to be a bit more specific to what you actually expect to be returned once foo is removed as I could change the removeItem function to set the value to 'somethingelse' and it'd still pass this test.